tests/python_tests/helpers/param_config.py

```diff
@@ -288,7 +288,7 @@ def format_combination(comb: tuple) -> str:
     return [format_combination(comb) for comb in all_params if comb[0] is not None]
 
 
-def input_output_formats(formats: List[DataFormat]) -> List[InputOutputFormat]:
+def input_output_formats(formats: List[DataFormat], same: Optional[bool]=False) -> List[InputOutputFormat]:
```

Comment:
```suggestion
def input_output_formats(formats: List[DataFormat], same: bool=False) -> List[InputOutputFormat]:
```
No point of having optional with default value :)

Reply:
Completely agree! 👍😀